lib/CLI.rb

```diff
@@ -5,88 +5,98 @@
 
 
 class CLI 
-    attr_reader :scraper
-    def initialize
-        @scraper = Scraper.new
-    end
+  attr_reader :scraper
+	
+  def initialize
+	@scraper = Scraper.new
+  end
 
-     def run
-      #  scraper = Scraper.new
-        scraper.getChamps
-        champion = Champion      
-        input = ""
-        while input!="exit"
-           
-            puts "Learn about each League of Legends Champions!"
-            puts "1: List Champion Classes and choose from them"
-            puts "2: Find a Champion by name"
-            puts "3: List ALL Champions"
-            puts "4: Exit"
-            #items, gameplay mechanics and terms might be a kind of useless in CLI 
-            
-            input = gets.strip.to_i
+  def run
+	scraper.getChamps
+	champion = Champion      
+	input = ""
+	while input!="exit"
+	  puts "| Learn about each League of Legends Champions! "
+	  puts "| 1: List Champion Classes and choose from them "
+	  puts "| 2: Find a Champion by name "
+	  puts "| 3: List ALL Champions"
+	  puts "| 4: Exit"
+	  #items, gameplay mechanics and terms might be a kind of useless in CLI 
+			
+	input = gets.strip.to_i
 
-            case input
-                when 1 
-                    puts "List Champion Classes and choose from them"
-                        champs_by_role
-                when 2
-                    puts "Find a champion by name"
-                        find_champ                    
-                when 3
-                    puts "List ALL champions by name"
-                        show_all_champs
-                when 4
-                    puts "exit"
-                    break
-            end
-        
-        end #end of while
-    end #end of run
+	case input
+	  when 1 
+		puts "List Champion Classes and choose from them"
+		champs_by_role
+	  when 2
+		puts "Find a champion by name"
+		find_champ                    
+	  when 3
+		puts "List ALL champions by name"
+		show_all_champs
+	  when 4
```

Comment:
Text doesn't follow the same capitalization format on each case. Champion vs champion on different lines, case `1` has classes capitalized, etc...